### PR TITLE
remove windows support

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

**Review by cfkoehler — APPROVED** [review]
Looks good to me. I searched through and could not find any other references to Windows

File: src/main/java/emissary/util/shell/Executrix.java [hunk 1]
```diff
@@ -848,28 +847,11 @@ public String[] getCommand(final String commandArg, final String[] tmpNames, fin
...
         c = c.replaceAll("<OUTPUT_NAME>", tmpNames[OUT]);
 
-        final String[] cmd;
-        if (System.getProperty("os.name").startsWith("Windows")) {
-            final int colPos = tmpNames[DIR].indexOf(":");
-            if (colPos > -1) {
-                // Naked Windows command shell
-                cmd =
-                        new String[] {"cmd", "/c",
-                                tmpNames[DIR].substring(0, colPos + 1) + " && cd " + tmpNames[DIR] + " && " + CYGHOME + "/bin/bash -c '" + c + "'"};
```
Comment (drivenflywheel) [inline src/main/java/emissary/util/shell/Executrix.java:858]: Line 60 which populates `CYGHOME` can also go:
```
public static final String CYGHOME = System.getProperty("CYGHOME", "c:/cygwin").replaceAll("\\Q\\\\E", "/");
```

**Review by drivenflywheel — CHANGES_REQUESTED** [review]
Overall looks good, just some straggler pieces to clean up.  Will be nice to have this gone.

File: src/main/java/emissary/util/shell/Executrix.java (hunk 1 above)
Comment (dev-mlb) [reply src/main/java/emissary/util/shell/Executrix.java:858]: done

File: src/main/java/emissary/pickup/file/FilePickUpClient.java [hunk 2]
```diff
@@ -39,15 +39,12 @@ public class FilePickUpClient extends emissary.pickup.PickUpSpace implements IPi
     protected WorkUnit currentWorkUnit = null;
 
     // These allow the same config file to drive this
-    // place on both windows and unix like systems where
-    // the path need to change based on the OS. For example
-    // we might have:
+    // place on unix like systems where the path need
```
[the remark this answers is not in the document]
Comment (dev-mlb) [reply src/main/java/emissary/pickup/file/FilePickUpClient.java]: done

**Review by ukengineer925 — DISMISSED** [review]

**Review by drivenflywheel — DISMISSED** [review]
LGTM - Adios, Windows!

**Review by drivenflywheel — APPROVED** [review]

**Review by cfkoehler — APPROVED** [review]

**Pull request merged**